build/charts/theia/README.md

```diff
@@ -30,14 +38,15 @@ Kubernetes: `>= 1.16.0-0`
 | clickhouse.service.tcpPort | int | `9000` | TCP port number for the ClickHouse service. |
 | clickhouse.service.type | string | `"ClusterIP"` | The type of Service exposing ClickHouse. It can be one of ClusterIP, NodePort or LoadBalancer. |
 | clickhouse.storage.createPersistentVolume.local.affinity | object | `{}` | Affinity for the Local Persistent Volume. By default it requires to label the Node used to store the ClickHouse data with "antrea.io/clickhouse-data-node=". |
+| clickhouse.storage.createPersistentVolume.local.nodes | list | `["kind-worker","kind-worker2"]` | A list of Node hostnames. Required when ClickHouse clustering is enabled and type is "Local". Please make sure to provide (shards * replicas) Nodes. Each Node should meet affinity and have the path created on it. |
 | clickhouse.storage.createPersistentVolume.local.path | string | `"/data"` | The local path. Required when type is "Local". |
-| clickhouse.storage.createPersistentVolume.nfs.host | string | `""` | The NFS server hostname or IP address. Required when type is "NFS". |
-| clickhouse.storage.createPersistentVolume.nfs.path | string | `""` | The path exported on the NFS server. Required when type is "NFS". |
+| clickhouse.storage.createPersistentVolume.nfs.hosts | list | `[]` | A list of NFS server hostname or IP address. Required when type is "NFS". When ClickHouse clustering is enabled, please provide (shards * replicas) NFS servers. Otherwise, only one NFS server is required. |
```

Comment:
I think we're not providing NFS servers but NFS shares, is that correct? They could be for instance all on the same server, mapped to locations assigned to different physical drives.

Reply:
An NFS share could be defined as `nfs.hosts:nfs.path`. Do you think this is better to let user input a list of `nfs.hosts:nfs.path` instead of accepting the two values separately?

Reply:
Updated as above, thanks